### DISCUSSION TITLE
Workaround to remove the default "v2" from CRUD endpoints

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#291] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi guys,

I was thinking about a workaround to remove the "v2" base path from the CRUD endpoints and I got the following idea: connect the endpoint to a "microservice" instead of "CRUD" and choose "crud-service" among the available microservices.
I have tried it and it seems to work. Do you see any drawback in this solution?

Thanks

**Comment by davidebianchi** [comment]
Hi @Lucascanna! Yes, the drawbacks are:
- if you use the cms, those new endpoints will not be available as pages
- the route are not listed in Console UI

Why do you need to remove it? We have the idea to remove the v2 base path in the future, only for the new crud endpoints.

**Reply by davidebianchi** [reply]
Those are the first two drawbacks, potentially there are others.

**Reply by fredmaggiowski** [reply]
I agree, you loose all the automatism done by the Console, if you are fine with that then you can define the endpoints as you wish 🤷

**Reply by theliuk** [reply]
[ 🤞 ] (https://makeitapp.atlassian.net/browse/MIAROAD-325)
